### PR TITLE
[#41897] Query disappearing from search and not visible

### PR DESCRIPTION
**Pull request by apfohl**
https://community.openproject.org/work_packages/41897

- Set `pageSize` for views API request to `-1` for sidebar
- Replaced deprecated Observable method

### CODE REVIEW

File: frontend/src/app/shared/components/op-view-select/op-view-select.component.ts [hunk 1]
```diff
@@ -121,10 +121,7 @@ export class ViewSelectComponent extends UntilDestroyedMixin implements OnInit {
       .onActivate(...this.menuItems)
       .subscribe(() => this.initializeAutocomplete());
 
-    this.views$ = combineLatest(
```
Comment (HDinger) [inline frontend/src/app/shared/components/op-view-select/op-view-select.component.ts:124]: We usually use the one-per-line style for two reasons: When making changes to the array it is better visible which observables have been removed/added. This also reduces possible merge conflicts later. Further, I think for those who work in a vim editor, it easier to grep those elements. Although, Ben is probably better in explain the concrete reasons for that last one 😅 
So simply adding the missing array brackets should be sufficient to make it look like this: 
```
this.views$ = combineLatest([
  this.search$,
  this.viewCategories$,
])
```

**Review by HDinger — COMMENTED** [review]
Good catch 👍  I only have one minor remark regarding code style

File: frontend/src/app/shared/components/op-view-select/op-view-select.component.ts (hunk 1 above)
Comment (apfohl) [reply frontend/src/app/shared/components/op-view-select/op-view-select.component.ts:124]: Then I suggest adding a rule to the linter or EditorConfig to enable auto-fixing. Remembering every rule in the head might be tedious. What do you think?

Comment (HDinger) [reply frontend/src/app/shared/components/op-view-select/op-view-select.component.ts:124]: Yes, good idea 👍

Comment (apfohl) [reply frontend/src/app/shared/components/op-view-select/op-view-select.component.ts:124]: I fixed the lines by hand. As I didn't got it to work automatically in eslint or edit config.

**Review by HDinger — APPROVED** [review]
👍 Good to merge once the tests are green

**Pull request merged**